docs/builds/guides/development/dll-builds-collaboration-features.md

```diff
@@ -0,0 +1,176 @@
+---
+menu-title: DLL builds for CKEditor 5 Collaboration Features
+category: builds-development
+order: 40
+modified_at: 2022-02-21
+---
+
+# CKEditor 5 DLL builds for CKEditor 5 Collaboration Features
+
+<info-box>
+    This guide discusses using a DLL build together with CKEditor 5 Collaboration Features and is supplemental to the {@link builds/guides/development/dll-builds CKEditor 5 DLL builds guide}.
+
+    Make sure to read the {@link builds/guides/development/dll-builds base guide} first before proceeding.
+</info-box>
+
+## Anatomy of a DLL build with CKEditor 5 collaboration features
+
+A DLL build of the editor consists of the following parts:
+
+* **Base DLL build**. It is a single JavaScript file that combines the contents of several core CKEditor 5 packages: `utils`, `core`, `engine`, `ui`, `clipboard`, `enter`, `paragraph`, `select-all`, `typing`, `undo`, `upload`, and `widget`. These packages are either the framework core, or are features used by nearly all editor installations (`ckeditor5` on npm).
+* **Base DLL build for CKEditor 5 Collaboration Features**. It is a single JavaScript file that includes all necessary files for the collaboration features packages and extends the base DLL for CKEditor 5 (`ckeditor5-collaboration` on NPM).
+* **DLL-compatible package builds**. Every package that is not a part of the base DLL builds, is built into a DLL-compatible JavaScript file (`@ckeditor/ckeditor5-*` on NPM). The CKEditor 5 Collaboration Features DLL builds are available in this format as well.
+
+In order to create an editor, you need to use the two base DLL builds plus several DLL-compatible package builds.
```

Comment:
```suggestion
In order to create an editor with collaboration features, you need to use the two base DLL builds plus a DLL-compatible package build for each plugin you would like to include.
```

Reply:
"a DLL-compatible package build for each plugin you would like to include" would be my suggestion rather.

Reply:
Hehe, I had it like this initially :).